initial-promises/node-server/inventory/1.0/test-inventory.pl

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/perl
```

Comment:
If it is not in the tools folder, then it should also be on the Technique folder, and you'll need to escape the & in the .st file